exercises/sum.cpp

```diff
@@ -9,19 +9,37 @@
 #include <iostream>
 using namespace std;
 
-int sum(int a, int b) {return a + b;}
+int getInput(){
+  int nInput;
+  cout << "Insert the addend: ";
+  cin >> nInput;
+  return nInput;
+}
+
+void setAddend(int & n, int & m){
+  n = getInput();
+  m = getInput();
+}
+
+int sumOperation(int a, int b) {return a + b;}
```

Comment:
i nomi verbosi potrebbero dare più info, ma in questo caso "sum" è più chiaro di "sumOperation" all'occhio umano